models/src/main/protobuf/DeployServiceV1.proto

```diff
@@ -43,6 +44,8 @@ service DeployService {
   rpc getBlocks(BlocksQuery) returns (stream BlockInfoResponse) {}
   // Find data sent to a name.
   rpc listenForDataAtName(DataAtNameQuery) returns (ListeningNameDataResponse) {}
+  // Find data sent to a par.
+  rpc getDataAtPar(DataAtParQuery) returns (RhoDataResponse) {}
```

Comment:
Please rename this method to `getDataAtName` and add comment to _listenForDataAtName_:
_OBSOLETE: Use getDataAtName instead. This method will be removed in the future version of RNode._

Reply:
@tgrospic, is comment `Find data sent to a par.` need to be updated to `Find data sent to a name.`?

Reply:
Resolved in [commit](https://github.com/rchain/rchain/pull/3592/commits/ee75c6842c3fb2bb872228c9f19c9dd96b84b024)

Reply:
Yes, makes sense.